### DISCUSSION TITLE
Vs code do not paste from clipboard

### DISCUSSION THREAD

**Discussion by Neoplayer** [end-4/dots-hyprland#721] [Q&A]
For some reason Arch vs code (visual-studio-code-bin + vscodium) does not paste text from the clipboard via ctrl+v if copied from a browser. I tried Brave and Firefox. But if I press **Super+v** and select what I copied, than it pastes it normally via Ctrl+v. I also tried copying from Gnome Text Editor. From it it pastes normally. 
This trouble only with code. Brave <-> Firefox <-> Text editor is ok.

**Comment by end-4** [comment]
is your vscode running in xwayland? 
to check, see if the window title on the overview is in italics

**Reply by Neoplayer** [reply]
`code --password-store=gnome --enable-features=UseOzonePlatform --ozone-platform=wayland`

![image](https://github.com/user-attachments/assets/1ae85068-3a6f-4266-80d6-2f28087b7aed)

**Reply by end-4** [reply]
it works now right?

**Reply by expl01txx** [reply]
still dont work

**Comment by Sunonymous** [comment]
I was experiencing this issue also. Eventually, I came across [this thread](https://stackoverflow.com/questions/78841771/clipboard-paste-not-working-in-vs-code-after-copying-from-chromium-based-browser), and the solution this person found worked for me as well. Give it a try!

**Comment by M4700F** [comment]
You can follow this. https://github.com/JaKooLit/Hyprland-Dots/discussions/384#discussioncomment-11239354
